velero-plugin-for-aws/object_store.go

```diff
@@ -284,6 +328,10 @@ func (o *ObjectStore) PutObject(bucket, key string, body io.Reader) error {
 	case o.kmsKeyID != "":
 		req.ServerSideEncryption = aws.String("aws:kms")
 		req.SSEKMSKeyId = &o.kmsKeyID
+	// if sseCustomerKey is not empty, assume SSE-C encryption with AES256 algorithm
```

Comment:
Is SSE-C encryption only supporting the AES256 algorithm?

Reply:
Yes, see the `x-amz-server-side​-encryption​-customer-algorithm` header at https://docs.aws.amazon.com/AmazonS3/latest/userguide/specifying-s3-c-encryption.html